task_sdk/src/airflow/sdk/definitions/asset/decorators.py

```diff
@@ -171,6 +171,7 @@ def create_dag(self, *, dag_id: str) -> DAG:
             dag_id=dag_id,
             schedule=self.schedule,
             is_paused_upon_creation=self.is_paused_upon_creation,
+            catchup=False,
```

Comment:
For schedule != None we need to either set this or provide start_date. This is better because we want to do start_date with partitions instead.